src/components/Quiz/Header.js

```diff
@@ -6,6 +6,7 @@ const Header = () => {
     return (
         <div className="quiz__header">
             <h3 className="quiz__header__question-number">{questionNumber !== 0 ? `${questionNumber}/${questionCount}` : ""}</h3>
+            <h3 className="quiz__header__score">Wynik: {player.score}</h3>
```

Comment:
Dodajmy https://react.i18next.com/ żeby przepuścić to przez system tłumaczeń :trollface: